src/components/Channels/Channels.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { fetchChannels } from "../../utils/fetcher";
+import Messages from "../Messages/Messages";
+
+class Channels extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      channels: [],
+      currentChannel: "General",
+      error: null
+    };
+  }
+
+  componentDidMount() {
+    let data = fetchChannels();
+    this.setState({ channels: data, currentChannel: "General" });
```

Comment:
```suggestion
    this.setState({ channels: data });
```

You already initialized it in the constructor don't need to do it again in the didMount().